### ISSUE TITLE
On-device symbolication is not working with Xcode 14 

### ISSUE THREAD

**Issue by YoloMao** [kstenerud/KSCrash#433]
When building project with Xcode 14，on-device symbolication is not working; Is it caused by _objc_msgSend stub or something else?

**Comment by YoloMao** [comment]
I found the problem of this error, when building with Xcode 14, symbol which n_type field equal to N_ENSYM will put the actual address to n_value field

**Comment by YoloMao** [comment]
```c
#define N_BNSYM 0x2e	/* begin nsect sym: 0,,n_sect,0,address */
#define N_ENSYM 0x4e	/* end nsect sym: 0,,n_sect,0,address */
```
I think we can simply filter N_BNSYM and N_ENSYM.

**Comment by YoloMao** [comment]
Solved at #434
